cmd/spiderpool-agent/cmd/daemon.go

```diff
@@ -234,6 +235,14 @@ func DaemonMain() {
 	}
 	agentContext.unixClient = spiderpoolAgentAPI
 
+	if agentContext.Cfg.EnableDRA {
+		logger.Info("Begin to start dra-plugin Server")
+		agentContext.DraPlugin, err = draplugin.StartDRAPlugin(logger, agentContext.Cfg.DraCdiRootPath, agentContext.Cfg.DraLibraryPath)
+		if err != nil {
+			logger.Fatal(err.Error())
+		}
+	}
```

Comment:
else {
    log: dra is closed
}

Reply:
why? else mean that the server has stared. L279 will log the server if it stop.

Reply:
tell that  the feature is disabled

Reply:
done, I thought you meant the service was shut down.